taskbench.c

```diff
@@ -43,10 +43,11 @@ int main(int argc, char **argv) {
 
     init(argc, argv);
 
-#ifdef OMPVER3
+//#ifdef OMPVER3
```

Comment:
Do we need to keep this and the similar ifdef's below commented out or remove them?

Reply:
I am not sure. That is why I commented it. We can just define it during compilation. I will revert this

Reply:
Reverting. Will add the macro in compilation command